AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -45,6 +45,8 @@ bloggingguidance.com##.row > center > h3
 myfin.by##.tg-discussion
 ! https://github.com/AdguardTeam/AdguardFilters/issues/75536
 jma.go.jp###jma_ad_config_button
+! https://github.com/AdguardTeam/AdguardFilters/issues/77294
+support.microsoft.com###uhf-upgrade-banner
```

Comment:
It can be closed, move it to the `popups.txt`.

Reply:
Okay.